### ISSUE TITLE
solarc depends of gnome-themes-standard, which does not exist

### ISSUE THREAD

**Issue by ploum** [regolith-linux/regolith-desktop#915]
regolith-look-solarized-dark depends of solarc-theme which, itself, depends of "gnome-themes-standard".

Problem is that gnome-themes-standard is not available in Debian bookworm.

https://packages.debian.org/search?keywords=gnome-themes-standard&searchon=names&suite=all&section=all

According to https://packages.ubuntu.com/search?keywords=gnome-themes-standard&searchon=names&suite=all&section=all it is not in Ubuntu 23.04.

This has been discussed in #635 but I believe that it should be its own bug.

**Comment by jbcom** [comment]
Same problem in Ubuntu Mantic, FYI

**Comment by rmdes** [comment]
Any update on this? 

apparently this has been fixed for Debian Testing but right now if you follow the doc in the context on Ubuntu 23 the installation of regolith desktop cannot complete, or even start actually.

**Comment by CypElf** [comment]
There is still no fix for this up to this day. Is there something planned or is this issue abandoned?

**Comment by kgilmer** [comment]
Hello, sorry responding to this thread slipped through the cracks.  This issue is fixed in Regolith 3.1:

```
$ cat /etc/apt/sources.list
deb http://deb.debian.org/debian/ bookworm main
deb-src http://deb.debian.org/debian/ bookworm main
...
$ sudo apt install regolith-look-solarized-dark 
Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
The following additional packages will be installed:
  arc-icon-theme faba-icon-theme fonts-nerd-font-meslo gnome-accessibility-themes gnome-icon-theme
  gnome-themes-extra gnome-themes-extra-data gruvbox-gtk gtk2-engines-murrine gtk2-engines-pixbuf
  librsvg2-2 librsvg2-common moka-icon-theme regolith-look-default-loader solarc-theme xrescat
Suggested packages:
  murrine-themes librsvg2-bin
The following NEW packages will be installed:
  arc-icon-theme faba-icon-theme fonts-nerd-font-meslo gnome-accessibility-themes gnome-icon-theme
  gnome-themes-extra gnome-themes-extra-data gruvbox-gtk gtk2-engines-murrine gtk2-engines-pixbuf
  librsvg2-2 librsvg2-common moka-icon-theme regolith-look-default-loader regolith-look-solarized-dark
  solarc-theme xrescat
0 upgraded, 17 newly installed, 0 to remove and 171 not upgraded.
Need to get 80.7 MB of archives.
After this operation, 231 MB of additional disk space will be used.
Do you want to continue? [Y/n]
```

3.1 is currently in beta, you can learn more here: https://regolith-desktop.com/docs/reference/Releases/regolith-3.1-release-notes/
